### ISSUE TITLE
Mismatch between behavior and rfc4254

### ISSUE THREAD

**Issue by fasmide** [fasmide/remotemoe#4]
This code tries to apply the same logic for both `exec` and `pty-req` requests. 
https://github.com/fasmide/remotemoe/blob/f745ed596a4ca6205b521c4fcea0cf45137ade0b/ssh/console.go#L33-L81

It's a mess, and it violates [6.5.  Starting a Shell or a Command](https://datatracker.ietf.org/doc/html/rfc4254#section-6.5) which states: 
```
   Once the session has been set up, a program is started at the remote
   end.  The program can be a shell, an application program, or a
   subsystem with a host-independent name.  Only one of these requests
   can succeed per channel.
```
Also, `exec` requests are less useful at the moment as the command line never exits on its own, reducing scriptability. 

For example, lets say a remote script just wants to know its own FQDN:
```
$ ssh remote.moe whoami
euhf ... ehfuih.remote.moe
<hang>
```
There is no good way of knowing when the command is finished.

**Comment by fasmide** [comment]
https://github.com/fasmide/remotemoe/compare/command-execution?expand=1#diff-7766c7b85ce331d0e9cb3228eea3a442e2ae903f00d08265493422aaa5808a53L45

While an easy fix, this still accepts both exec and pty-req on the same channel which the RFC forbids

seems to work rather well though...

**Comment by fasmide** [comment]
Fixed in #5
